README.md

```diff
@@ -6,12 +6,12 @@
 
 **The Unity Machine Learning Agents Toolkit** (ML-Agents) is an open-source Unity plugin 
 that enables games and simulations to serve as environments for training
-intelligent agents. Agents can be trained using reinforcement learning,
```

Comment:
Agents should not be capitalized.

Reply:
reverted back and will go through this again